chore(documentation): a bit of redesign (VIV-000)

rachelbt opened the pull request.

rachelbt commented:
@AyalaBu , @YonatanKra @olaf-k 
WDTY?
https://vivid-vmzmkwvrn3hg.deno.dev/

AyalaBu commented:
@rachelbt 
It looks nice
Maybe adding the image we spoke about earlier will add to it

YonatanKra commented on apps/docs/assets/styles/main.css:
```diff
@@ -52,13 +52,13 @@ vwc-side-drawer::part(base) {
-vwc-header[elevation-shadow] vwc-side-drawer::part(base) {
-	border-right: 1px solid var(--vvd-color-neutral-100);
-}
+/*vwc-header[elevation-shadow] vwc-side-drawer::part(base) {*/
+/*	border-right: 1px solid var(--vvd-color-neutral-100);*/
+/*}*/
```
If this is not needed, remove :)

rachelbt replied:
```suggestion

```

rachelbt commented:
> if I set the theme to be dark I don't want a large fixed white band at the top of the screen :(

but that's how our dark-light theme works :(

rinaok commented:
@rachelbt @AyalaBu  NICE!!! Looks good :)

AyalaBu commented:
@rinaok 
I wonder: maybe we can try giving the header the same color as the side-nav?
Because right now it is too much and is really overwhelming

rinaok reviewed (changes requested):
<img width="1783" alt="Screenshot 2023-06-21 at 10 33 36" src="https://github.com/Vonage/vivid-3/assets/10883919/d7af69a3-5c73-43e9-9545-43877275219f">


In foundations page the header is not aligned

rachelbt commented:
> <img alt="Screenshot 2023-06-21 at 10 33 36" width="1783" src="https://user-images.githubusercontent.com/10883919/247467986-d7af69a3-5c73-43e9-9545-43877275219f.png">
> In foundations page the header is not aligned

there was an un-needed space before the headline. removed it :)

rinaok commented:
> > <img alt="Screenshot 2023-06-21 at 10 33 36" width="1783" src="https://user-images.githubusercontent.com/10883919/247467986-d7af69a3-5c73-43e9-9545-43877275219f.png">
> > In foundations page the header is not aligned
> 
> there was an un-needed space before the headline. removed it :)

![Screenshot 2023-06-21 at 13 46 49](https://github.com/Vonage/vivid-3/assets/10883919/47bb3d4d-1164-47f3-ae44-454f1f4f1692)


@rachelbt now it looks like this lol

rachelbt commented:
> > > <img alt="Screenshot 2023-06-21 at 10 33 36" width="1783" src="https://user-images.githubusercontent.com/10883919/247467986-d7af69a3-5c73-43e9-9545-43877275219f.png">
> > > In foundations page the header is not aligned
> > 
> > 
> > there was an un-needed space before the headline. removed it :)
> 
> ![Screenshot 2023-06-21 at 13 46 49](https://user-images.githubusercontent.com/10883919/247522931-47bb3d4d-1164-47f3-ae44-454f1f4f1692.png)
> 
> @rachelbt now it looks like this lol

there was something strange in the readme file. maybe because it was created outside of an IDE.
Fixed it with css :)
